orchagent/portsorch.cpp

```diff
@@ -39,6 +39,37 @@ static map<string, sai_port_fec_mode_t> fec_mode_map =
     { "fc", SAI_PORT_FEC_MODE_FC }
 };
 
+static const vector<sai_port_stat_t> portStatIds =
```

Comment:
static is not required here.
you may check the links below for explanations
https://stackoverflow.com/questions/11055802/static-and-extern-global-variables-in-c-and-c
http://en.cppreference.com/w/cpp/language/storage_duration